clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -322,6 +328,18 @@ private void updateTask(StreamingCacheState state, Task task) {
       state.changes.incrementAndGet();
     }
 
+    private void updateMovedTask(StreamingCacheState state, Task task, String destinationJobId,
+                                 String sourceJobId) {
+      if (sourceJobId != null) {
+        if (state.tasks.containsKey(sourceJobId)) {
+          state.tasks.get(destinationJobId).add(task);
```

Comment:
Since `updateTask()` is still called after this method, you don't need to do anything wrt the destination job here. Can remove this line, as well as the `state.updatedJobs.add(destinationJobId)`.

Reply:
fixed